librz/include/rz_types_overflow.h

```diff
@@ -5,9 +5,21 @@
 
 // TODO: Use CLANG/GCC builtins if available: __builtin_mul_overflow
 
-// ADD
-// if ((x > 0) && (a > INT_MAX - x)) /* `a + x` would overflow */;
-// if ((x < 0) && (a < INT_MIN - x)) /* `a + x` would underflow */;
+//Using CLANG/GCC builtins
```

Comment:
Space before `Using`. Also, it's probably better to write "Using compiler builtins when available"